src/main/java/data/Athlete.java

```diff
@@ -0,0 +1,31 @@
+package data;
+
+import java.math.BigDecimal;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class Athlete implements Comparable<Athlete> {
+    private final String name;
+    private Map<String, String> events = new LinkedHashMap<>();
+
+    public Athlete(String name, Map<String, String> events) {
+        this.name = name;
+        this.events = events;
+    }
+
+    public Map<String, String> getEvents() {
+        return events;
+    }
+    public String getName() {
+        return name;
+    }
+
+    @Override
+    public int compareTo(Athlete a) {
+        return getTotalPoints().compareTo(a.getTotalPoints());
+    }
+
+    public BigDecimal getTotalPoints() {
+        return events.entrySet().stream().map(e -> Event.fromString(e.getKey()).get().getPoints(e.getValue())).reduce(BigDecimal.ZERO, (x, y) -> x.add(y));
```

Comment:
Oneliner
If you like oneliners, you can check out my oneliner. As you can see, the comment is as long as the oneliner.
Code:
```
#!/bin/bash
# recursively finds all git repos (2 levels deep) and executes given git command on them
# omits .ws .git and .idea directories
# example: bash gitAll.sh pull
# result: pull from all repos

find . -not -regex '.*\(.git\|.idea\|.ws\).*' \
  -maxdepth 2 -mindepth 2 -type d \
  -exec echo {} \; \
  -exec git --git-dir={}/.git --work-tree=$PWD/{} $@ \;

```